File: module_mine/src/main/java/com/yhklsdf/module_mine/MineActivity.java
package com.yhklsdf.module_mine;

import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MineActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_mine);
        FragmentManager manager = getSupportFragmentManager();
        FragmentTransaction transaction = manager.beginTransaction();
        transaction.replace(R.id.framelayout, new MineFragment());
        transaction.commit();
    }
}

File: module_mine/src/main/java/com/yhklsdf/module_mine/MineActivity.kt
package com.yhklsdf.module_mine

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.yhklsdf.module_mine.R

class MineActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_mine)
        supportFragmentManager.beginTransaction().replace(R.id.container, MineFragment()).commit()
    }
}
